micro_ros_setup/config/zephyr/generic/create.sh

```diff
@@ -0,0 +1,52 @@
+# To be removed later, please use package.xml
+sudo apt update
+sudo apt install --no-install-recommends git cmake ninja-build gperf ccache dfu-util device-tree-compiler wget python3-pip python3-setuptools python3-tk python3-wheel xz-utils file make gcc gcc-multilib software-properties-common -y
+
+wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | sudo apt-key add -
+sudo apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main'
+sudo apt update
+sudo apt install cmake -y
+
+# pip3 install --user -U west
+export PATH=~/.local/bin:"$PATH"
+
+pushd $FW_TARGETDIR >/dev/null
+   
+    west init zephyrproject
+    pushd zephyrproject >/dev/null
+        west update
+    popd >/dev/null
+
+    pip3 install -r zephyrproject/zephyr/scripts/requirements.txt
+
+    export TOOLCHAIN_VERSION=zephyr-toolchain-arm-0.11.2-setup.run
+    wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.11.2/$TOOLCHAIN_VERSION
+    chmod +x $TOOLCHAIN_VERSION
+    ./$TOOLCHAIN_VERSION -- -d $(pwd)/zephyr-sdk -y
+
+    rm -rf $TOOLCHAIN_VERSION
+
+
+    # Temporal until driver in mainstream
+    pushd zephyrproject/zephyr >/dev/null
+        git remote add eprosima https://github.com/eProsima/zephyr
```

Comment:
This should be in a repos file.

Reply:
This is a temporal checkout of one of the repos downloades by west because we are waiting the merge of one PR on the Zephyr upstream. Once it is merged this will be removed.